timeseries/src/autogluon/timeseries/models/statsmodels/abstract_statsmodels.py

```diff
@@ -0,0 +1,175 @@
+        default_model_init_args, default_model_fit_args = self._get_default_model_init_and_fit_args()
+        # TODO: Should we rather set n_jobs in __init__?
+        self.n_jobs = default_model_fit_args.pop("n_jobs", -1)
+
+        # Fit models in parallel
+        fit_fn = partial(
+            self._fit_local_model,
+            default_model_fit_args=default_model_fit_args,
+            default_model_init_args=default_model_init_args,
+        )
+        fitted_models = Parallel(n_jobs=self.n_jobs)(delayed(fit_fn)(timeseries=ts) for ts in timeseries_to_fit)
+        for item_id, model in zip(items_to_fit, fitted_models):
+            self._fitted_models[train_hash.loc[item_id]] = model
+
+    def predict(self, data: TimeSeriesDataFrame, quantile_levels: List[float] = None, **kwargs) -> TimeSeriesDataFrame:
+        self._check_predict_inputs(data, quantile_levels=quantile_levels, **kwargs)
+        if quantile_levels is None:
+            quantile_levels = self.quantile_levels
+
+        # Make sure that we fitted a local model to each time series in data
+        data_hash = hash_ts_dataframe_items(data)
+        items_to_fit = [item_id for item_id, ts_hash in data_hash.iteritems() if ts_hash not in self._fitted_models]
+        if len(items_to_fit) > 0:
+            logger.info(f"{self.name} received {len(items_to_fit)} items not seen during training, re-running fit")
+            self._fit(train_data=data)
+
+        # Make predictions in parallel
+        predict_fn = partial(self._predict_with_local_model, quantile_levels=quantile_levels)
+        timeseries_with_models = (
+            (data.loc[item_id][self.target], self._fitted_models[ts_hash])
+            for item_id, ts_hash in data_hash.iteritems()
+        )
+        predictions = Parallel(n_jobs=self.n_jobs)(
+            delayed(predict_fn)(timeseries=ts, fitted_model=model) for ts, model in timeseries_with_models
+        )
+
+        # Combine all predictions into a single DataFrame
+        result_df = pd.concat({item_id: pred for item_id, pred in zip(data.iter_items(), predictions)})
+        result_df.index.rename([ITEMID, TIMESTAMP], inplace=True)
+        return TimeSeriesDataFrame(result_df)
+
+    def _predict_with_local_model(
+        self, timeseries: pd.Series, fitted_model: FittedLocalModel, quantile_levels: List[float]
+    ) -> pd.DataFrame:
+        raise NotImplementedError
+
+    def _get_predictions_from_initialized_model(self, initialized_model, cutoff, quantile_levels):
+        start = cutoff + pd.tseries.frequencies.to_offset(self.freq)
+        end = cutoff + self.prediction_length * pd.tseries.frequencies.to_offset(self.freq)
+        predictions = initialized_model.get_prediction(start=start, end=end)
+        results = [predictions.predicted_mean.rename("mean")]
+        for q in quantile_levels:
+            if q < 0.5:
+                coverage = 2 * q
```

Comment:
does this work? doesn't `q=0.5` result in `alpha/coverage=1` ?

Reply:
`alpha=1` computes that median, so that's what we need